avro/src/schema.rs

```diff
@@ -1353,6 +1358,7 @@ impl Parser {
         &mut self,
         complex: &Map<String, Value>,
         enclosing_namespace: &Namespace,
+        from_field: bool,
```

Comment:
IMO it would be better to use an `enum` with well named variants instead of `true` / `false`

Reply:
Great idea! I'm updating it